src/test/java/Homework16.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+        @Test
+    public static void RegistrationButton() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement RegistrationButton = driver.findElement(By.id("hel"));
+        RegistrationButton.click();
+        String url2 = "https://bbb.testpro.io/registration.php";
```

Comment:
This variable should be renamed to something more descriptive , like registrationUrl